### ISSUE TITLE
Bundled pgAdmin4 psycopg2 error after login

### ISSUE THREAD

**Issue by barzog** [CrunchyData/postgres-operator#3367]
We setup simple cluster (1 replica/1instance/2 databases and 2 users ) on 1 node k3s using latest postgresql-operator-examples. Cluster itself works well. But pgAdmin4 using following reference:
  userInterface:
    pgAdmin:
      image: registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi8-4.30-3
      dataVolumeClaimSpec:
        accessModes:
        - "ReadWriteOnce"
        resources:
          requests:
            storage: 1Gi
gives us following error after user login on connect to any database defined: “'psycopg2.extensions.Column' object has no attribute '_asdict”

Deploying pgAdmin 6.13 from https://helm.runix.net/ works as expected. Is anyone using bundled pgAdmin4?

**Comment by sskirito** [comment]
I have the same error. Have you got any solutions

**Comment by barzog** [comment]
> I have the same error. Have you got any solutions

Yes. I've deployed other pgAdmin4 version through helm:
_helm repo add runix https://helm.runix.net/
helm repo update_
Created _pgadmin4_values.yaml_:
```
---
env:
  email: <admin email>
  password: <admin password>
persistentVolume:
  size: 1Gi
image:
  tag: 6.13
serverDefinitions:
  enabled: true
  servers:
    <DB name from PGO>:
      Name: "<Some DB Name>"
      Group: "<Some DB Group>"
      Port: 5432
      Username: "<username from pgo>"
      Host: "<hostname from pgo>"
      SSLMode: "prefer"
      MaintenanceDB: "<DB Name>"
```
Created that to new namespace: _helm install pgadmin4 runix/pgadmin4 --create-namespace --namespace pgadmin4 -f pgadmin4-values.yaml_
added new ingressroute to traefik and voila.

**Comment by tjmoore4** [comment]
@barzog This has been addressed with [crunchydata/crunchy-containers#1480](https://github.com/CrunchyData/crunchy-containers/pull/1480) and is planned for inclusion in the next release. Until then, you can downgrade to tag `ubi8-4.30-2` to avoid this issue.
